airflow/hooks/base.py

```diff
@@ -59,6 +60,20 @@ def get_connection(cls, conn_id: str) -> Connection:
         :param conn_id: connection id
         :return: connection
         """
+        # TODO: This is not the best way of having compat, but it's "better than erroring" for now. This still
+        # means SQLA etc is loaded, but we can't avoid that unless/until we add import shims as a big
+        # back-compat layer
+
+        # If this is set it means are in some kind of execution context (Task, Dag Parse or Triggerer perhaps)
+        # and should use the Task SDK API server path
+        if hasattr(sys.modules.get("airflow.sdk.execution_time.task_runner"), "SUPERVISOR_COMMS"):
+            # TODO: AIP 72: Add deprecation here once we move this module to task sdk.
+            from airflow.sdk import Connection as TaskSDKConnection
+
+            return TaskSDKConnection.get(
+                conn_id=conn_id,
+            )
```

Comment:
Nit
```suggestion
            return TaskSDKConnection.get(conn_id=conn_id)
```

(The trailing comma makes ruff formatt it on it's own line)

Reply:
Cool yeah